src/api/server/sync_io.rs

```diff
@@ -121,7 +121,11 @@ impl<F: FileSystem + Sync> Server<F> {
 
     fn lookup<S: BitmapSlice>(&self, mut ctx: SrvContext<'_, F, S>) -> Result<usize> {
         let buf = ServerUtil::get_message_body(&mut ctx.r, &ctx.in_header, 0)?;
```

Comment:
Seems all get_message_body() calls and other similar calls could just error out without replying to kernel as well?